### ISSUE TITLE
3.10.0-dev.632:error of NavigationView in Skia.Wpf project

### ISSUE THREAD

**Issue by ywmoyue** [unoplatform/uno#6984]
### Current behavior

When I update Unoui nuget package from 3.9.7 to 3.10.0-dev.632, NavigationView in SkiaWpf project has an error occurred.

![image](https://user-images.githubusercontent.com/28388300/132351796-cc6f9fae-405d-4b4a-8e7b-5e3f9c559937.png)

It looks like this on version 3.9.7
![image](https://user-images.githubusercontent.com/28388300/132352348-c670b3e1-0b15-4749-9504-b6bd902574cd.png)

### How to reproduce it (as minimally and precisely as possible)

MainPage.xaml : 
```xml
<Page
    x:Class="uApp3.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:uApp3"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
    mc:Ignorable="d">
    <Page.DataContext>
        <local:MainPageViewModel x:Name="viewModel"/>
    </Page.DataContext>
    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
		<muxc:NavigationView x:Name="mainNavView"
                            PaneDisplayMode="Left"
                             MenuItemsSource="{Binding NavMenu}"
                            IsSettingsVisible="False">
            <muxc:NavigationView.MenuItemTemplate>
                <DataTemplate>
                </DataTemplate>
            </muxc:NavigationView.MenuItemTemplate>
            <Frame x:Name="mainFrame">
            </Frame>
        </muxc:NavigationView>
    </Grid>
</Page>
```

MainPageViewModel.cs : 
```cs
using System.Collections.ObjectModel;

namespace uApp3
{
    public class MainPageViewModel
    {
        public ObservableCollection<string> NavMenu { get; set; }
        public MainPageViewModel()
        {
            NavMenu = new ObservableCollection<string> { "test1", "test2", "test3" };
        }
    }
}
```

### NuGet package version(s)

- Uno.UI.RemoteControl 3.10.0-dev.632
- Uno.UI.Skia.Wpf 3.10.0-dev.632
- Microsoft.Extensions.Logging 5.0.0
- Microsoft.Extensions.Logging.Console 5.0.0
- NETStandard.Library 2.0.3

### Affected platforms

Skia (WPF)

### IDE

Visual Studio 2019

### IDE version

Microsoft Visual Studio Community 2019 16.11.2 Preview 1.0

**Comment by jeromelaban** [comment]
Likely fixed by https://github.com/unoplatform/uno/pull/6956, wait for 3.10.0-dev.634 to validate again, thanks!

**Comment by ywmoyue** [comment]
> Likely fixed by #6956, wait for 3.10.0-dev.634 to validate again, thanks!

I updated to 3.10.0-dev.647 and the error disappeared, thank you
